Lib/test/test_doctest.py

```diff
@@ -1194,7 +1194,7 @@ def displayhook(): r"""
 Test that changing sys.displayhook doesn't matter for doctest.
```

Comment:
I think the whole test should be rewritten to test that doctest uses the custom user-defined `displayhook`.